features/step_definitions/kata.rb

```diff
@@ -1,16 +1,88 @@
-require 'pry-byebug'
 Given /^kata container has been installed successfully(?: in the #{QUOTED} project)?$/ do |ns|
-  binding.pry
   kata_ns ||= "kata-operator"
   step %Q/I switch to cluster admin pseudo user/
   unless namespace(kata_ns).exists?
     @result = user.cli_exec(:create_namespace, name: kata_ns)
     raise "Failed to create namespace #{kata_ns}" unless @result[:success]
   end
   project(kata_ns)
-  # setup service account
-  role_yaml = "https://raw.githubusercontent.com/openshift/kata-operator/release-4.6/deploy/role.yaml"
-  role_binding_yaml = "https://raw.githubusercontent.com/openshift/kata-operator/release-4.6/deploy/role_binding.yaml"
-  sa_yaml = "https://raw.githubusercontent.com/openshift/kata-operator/release-4.6/deploy/service_account.yaml"
+  iaas_type = env.iaas[:type] rescue nil
+  raise "Kata installation only supports GCE platform currently." if iaas_type != 'gcp'
```

Comment:
Actually I think Azure works even better, it has nested virt enabled and you can actually run a kata container. Maybe not necessary for testing the installation only.

Reply:
@jensfr ok, I have not tested with Azure yet.  I'll give it a try...if it works, we can filter out more laterr

Reply:
@jensfr I've verified that Azure works as well.  Updated the code to filter from a list of acceptable platforms.